utils/update_report_submission.R

```diff
@@ -0,0 +1,33 @@
+# Packages ----------------------------------------------------------------
+
+require(here)
+require(stringr)
+require(dplyr)
+require(purrr)
+
+# Control parameters ------------------------------------------------------
+
+forecast_date <- Sys.Date()
+forecast_dir <- here::here("rt-forecast")  # Assumes forecasts are in national and regional subfolders
+
+# Find forecasts ----------------------------------------------------------
+
+forecasts <- c(list.dirs(file.path(forecast_dir, "regional"), recursive = FALSE),
+               list.dirs(file.path(forecast_dir, "national"), recursive = FALSE))
+names(forecasts) <- forecasts %>% 
+  stringr::str_remove(file.path(forecast_dir, "regional/")) %>% 
+  stringr::str_remove(file.path(forecast_dir, "national/"))
+
+# Extract forecasts -------------------------------------------------------
+
+source(here::here("utils/format_submission.R"))
+
+region_forecasts <- purrr::map2_dfr(forecasts, names(forecasts),
+                                    ~ format_forecast(loc = .x, loc_name = .y, 
+                                                      forecast_date = forecast_date,
+                                                      forecast_adjustment = 5 + 9))
+
+# Save forecast -----------------------------------------------------------
+
+readr::write_csv(region_forecasts, 
```

Comment:
saving as an ensemble when not an ensemble? if for there processes do we also need to say will be ensemble in submission stuff?